mgmt/src/models/internal/natconfig/table_extend.rs

```diff
@@ -3,10 +3,11 @@
 
 //! NAT rule tables entries creation
 
-use crate::models::external::overlay::vpc::Peering;
+use crate::models::external::overlay::vpc::{Peering, VpcTable};
```

Comment:
As discussed we will move the destination vni selection logic to its own pipeline stage at a future date, again please correct me if I am mistaken.

Reply:
You're not mistaken. But until we have that stage, we need this to be done by the nat code.